lib/cylc/gui/app_gcylc.py

```diff
@@ -548,11 +550,7 @@ def __init__(self, suite, db, owner, host, port, pyro_timeout,
         bigbox.pack_start(hbox, False)
 
         self.window.add(bigbox)
-        title = "gcylc"
-        if self.restricted_display:
-            title += " -r (restricted display)"
-        self.window.set_title(title)
-
+        self.window.set_title('gcylc - %s' % self.cfg.my_uuid)
```

Comment:
I really don't like the UUID in the gcylc window title - can it be moved to the 'About' dialog or something?